airflow/providers/amazon/aws/sensors/emr.py

```diff
@@ -424,12 +431,16 @@ def __init__(
         job_flow_id: str,
         target_states: Iterable[str] | None = None,
         failed_states: Iterable[str] | None = None,
+        deferrable: bool = False,
```

Comment:
`deferrable` is the latest argument in many of the operators. Not sure whether we want to keep that behavior

Reply:
Not sure if I understand it. Can you please elaborate a little bit

Reply:
Sure! What I mean is whether we should exchange the order of the newly added arguments. As other operator with deferrable functionality seem to be implemented this way

```suggestion
        max_attempts: int = 60,
        deferrable: bool = False,
```

Reply:
I'm not sure If we followed such a convention but making deferrable param consistent makes sense  https://github.com/search?q=repo%3Aapache%2Fairflow+deferrable%3A+bool+path%3A%2F%5Eairflow%5C%2Fproviders%5C%2Famazon%5C%2Faws%5C%2F%2F&type=code